iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/HvdcLineAdderAdapter.java

```diff
@@ -0,0 +1,180 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.mergingview;
+
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.iidm.network.HvdcLine;
+import com.powsybl.iidm.network.HvdcLineAdder;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.util.Identifiables;
+
+import java.util.Objects;
+
+/**
+ * @author Thomas Adam <tadam at silicom.fr>
+ */
+public class HvdcLineAdderAdapter implements HvdcLineAdder {
```

Comment:
Why the design of this adder is completely different from the others: why don't we have a delegate here?

Reply:
I think, the delegate cannot be used here
Which network will be used to create the HvdcLine ?
The network to use cannot be known before `setConverterStationId` methods

Reply:
Right... My bad, it miss something when I did the review